server/README.md

```diff
@@ -136,3 +136,9 @@ The easiest way to get these tests to pass is to:
 Alternatively, if you're only interested in running unit tests, you can just run `cargo test --lib`. These tests don't make any assumptions about the surrounding environment.
 
 To run only a specific test (e.g. only the application tests), you can use the `--test` flag to `cargo test` which supports common Unix glob patterns. For example: `cargo test --test '*app*'`.
+
+# Code Coverage
+To get a code coverage report, first make sure you have the following installations:
+- `cargo install grcov`
+- `rustup component add llvm-tools-preview`
```

Comment:
This won't work if default is not set to nightly. Probably just need to say "use nightly" which will also simplify the rest.

Reply:
I ran both of these install commands on stable successfully

Reply:
Yeah, they install fine, but I had issues. Though maybe my issues were unrelated.